apps/api/migrations/subscriber-preferences-level/subscriber-preferences-level.migration.ts

```diff
@@ -0,0 +1,33 @@
+import '../../src/config';
+
+import { NestFactory } from '@nestjs/core';
+import { PreferenceLevelEnum, SubscriberPreferenceRepository } from '@novu/dal';
+
+import { AppModule } from '../../src/app.module';
+
+export async function addLevelPropertyToSubscriberPreferences() {
+  // eslint-disable-next-line no-console
+  console.log('start migration - add level property to subscriber preferences');
+  const app = await NestFactory.create(AppModule, {
+    logger: false,
+  });
+
+  const subscriberPreferenceRepository = app.get(SubscriberPreferenceRepository);
+  // eslint-disable-next-line no-console
+  console.log('add level: PreferenceLevelEnum.TEMPLATE to all subscriber preferences without level property');
+
+  await subscriberPreferenceRepository._model.collection.updateMany(
+    { level: { $exists: false }, _templateId: { $exists: true } },
+    {
+      $set: { level: PreferenceLevelEnum.TEMPLATE },
+    }
+  );
```

Comment:
currently all the preference have templateId and will be converted to level: template